src/asr/asr_chainer.py

```diff
@@ -52,12 +52,11 @@
 
 
 class ChainerSeqEvaluaterKaldi(extensions.Evaluator):
```

Comment:
Now you don't have to pass "reader" and you can use built-in `chainer.training.extensions.Evaluator` instead of making our own Evaluator.